packages/calcite-components/src/components/alert/alert.scss

```diff
@@ -26,17 +41,22 @@ $border-style: 1px solid var(--calcite-color-border-3);
   my-0
   opacity-0
   pointer-events-none
-  shadow-2
   text-start
   w-full
   z-toast;
 
-  border-radius: var(--calcite-border-radius);
+  --calcite-internal-alert-border-color: var(--calcite-alert-border-color, var(--calcite-color-border-3));
+  --calcite-internal-alert-corner-radius: var(--calcite-alert-corner-radius, var(--calcite-border-radius));
+
+  background-color: var(--calcite-alert-background-color, var(--calcite-color-foreground-1));
+  border-radius: var(--calcite-internal-alert-corner-radius, var(--calcite-border-radius));
```

Comment:
I can't see anywhere the corner radius is redefined. I think this should be `--calcite-alert-corner-radius`?

Reply:
Not redefined, but reused.